src/test/scala/ie/zalando/fabric/gateway/service/IngressDerivationChainSpec.scala

```diff
@@ -629,6 +640,7 @@ class IngressDerivationChainSpec extends FlatSpec with MockitoSugar with Matcher
     val routeFilters = routes
       .filterNot(isCatchAllRoute)
       .filterNot(isHttpRejectRoute)
+      .filterNot(isCorsRoute)
```

Comment:
This seems like a very brittle test. Filter out some things, then verify there are things left! I know it's not officially part of this PR, but can it be addressed?